appveyor.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.0.{build}
+
+skip_tags: true
+
+skip_branch_with_pr: true

```

Comment:
Does this mean we won't get any AppVeyor builds for PRs coming from a forked repo?

Reply:
It means AppVeyor won't build both the branch *and* the PR merge. Once a PR is created, AppVeyor will only build the PR merge.